### ISSUE TITLE
[Feature]: i18n

### ISSUE THREAD

**Issue by brolga** [sveltinio/sveltin#118]
Do you have plans to support multiple language for being able to switch between languages on the Website ?

Kind Regards,
Burkhard

**Comment by indaco** [comment]
Yes, it is in the backlog but needs to be evaluated from different angles. 

If you have ideas on how to support it, more than welcome to hear from you.

Meanwhile you could have a look to some i18n libs like https://github.com/sveltekit-i18n/lib and use it in your sveltin projects. 

Thanks
